### PR TITLE
Keith dev

### PR DESCRIPTION
**Pull request by mwaloszek55**

### CODE REVIEW

File: game/characters/nomad.py [hunk 1]
```diff
@@ -51,7 +62,7 @@ def attack(self, surface, target):
                     self.projectile_sound.play()
                     self.projectiles.append(
                         Projectile(self.rect.centerx - (2 * self.rect.width * self.flip), self.rect.y, 2 * self.rect.width,
-                                    self.rect.height // 2, damage, knockback, self, 10 - (20 * self.flip)))
+                                    self.rect.height // 2, damage, knockback, self, 10 - (20 * self.flip), tornado))
```
Comment (michaelc445) [inline game/characters/nomad.py:65]: is this loading the projectile image from disk each time one is created?

File: game/characters/wizard.py [hunk 2]
```diff
@@ -35,7 +45,7 @@ def attack(self, surface, target):
                     self.projectile_sound.play()
                     self.projectiles.append(
                         Projectile(self.rect.centerx - (2 * self.rect.width * self.flip), self.rect.y, self.rect.width,
-                                self.rect.height // 2, damage, knockback, self, 10 - (20 * self.flip)))
+                                self.rect.height // 2, damage, knockback, self, 10 - (20 * self.flip), pulse))
```
Comment (michaelc445) [inline game/characters/wizard.py:48]: same here, we should probably just load the images once and pass the scaled version to the projectile constructor

File: game/characters/wizard.py [hunk 3]
```diff
@@ -46,7 +56,7 @@ def attack(self, surface, target):
                     self.projectile_sound.play()
                     self.projectiles.append(
                         Projectile(self.rect.centerx - (2 * self.rect.width * self.flip), self.rect.y, 2 * self.rect.width,
-                                self.rect.height, damage, knockback, self, 5 - (10 * self.flip)))
+                                self.rect.height, damage, knockback, self, 5 - (10 * self.flip), orbSpell))
```
Comment (michaelc445) [inline game/characters/wizard.py:59]: same here

File: game/fighter.py [hunk 4]
```diff
@@ -41,6 +45,16 @@ def loadImages(self, spriteSheet, animationSteps):
                 tempImageList.append(tempImage)
             animationList.append(tempImageList)
             y += 1
+        return animationList#
```
Comment (michaelc445) [inline game/fighter.py:48]: animationList# ?

**Review by michaelc445 — CHANGES_REQUESTED** [review]

**Comment by Konrad-Dagiel** [comment]
Works for me

**Review by michaelc445 — APPROVED** [review]

**Review by Konrad-Dagiel — APPROVED** [review]
works for me nice

**Pull request merged**